components/DocumentationNavigation/DocumentationNavigation.tsx

```diff
@@ -1,4 +1,4 @@
-import { useState } from 'react'
+import React, { useState } from 'react'
```

Comment:
Why are we importing React? We aren't using it anywhere as far as I can tell.

Reply:
I think you're supposed to for any file with jsx